static/src/javascripts/projects/journalism/modules/render-campaign.js

```diff
@@ -16,16 +16,21 @@ const renderCampaign = (anchorNode: HTMLElement): void => {
             anchorNode.insertAdjacentHTML('afterend', campaignDiv);
         })
         .then(() => {
-            const cForm = anchorNode.querySelector('.campaign form');
+            const cForm = document.querySelector(
+                '.element-campaign .campaign .campaign--snippet__body'
+            );
             if (cForm) {
                 cForm.addEventListener('submit', submitForm);
             }
         });
 };
 
 export const initCampaign = () => {
+    const calloutData = getCampaign();
+
     const fourthParagraph = document.querySelector(
         '.content__article-body p:nth-of-type(4)'
     );
-    if (fourthParagraph) renderCampaign(fourthParagraph);
+    if (calloutData && fourthParagraph)
```

Comment:
Does this mean that the campaign shouldn't be rendered if there are 3 paragraphs and a figure?

Reply:
For now yes. Though good point. 
The logic for the location on the page is likely to change to place it the end of the story, and this is one of the reasons why.